pydictable/core.py

```diff
@@ -136,6 +142,7 @@ def __apply_dict(self, d: dict):
             value = d.get(self.get_field_key(attr))
             if not field.required and value is None:
                 continue
+            field.config = self._config
```

Comment:
Instead of assigning it, can we pass it while running the field? To from_dict or to_dicts?